generics.R

```diff
@@ -10,11 +10,6 @@ vetiver_create_meta._lgb.Booster <- function(model, metadata) {
 }
 
 
-handler_startup._lgb.Booster <- function(vetiver_model) {
-  attach_pkgs(vetiver_model$metadata$required_pkgs)
-}
```

Comment:
Turns out `vetiver_create_description` and `vetiver_create_meta` _are_ in fact necessary for calling `vetiver_model` on the model, so I added them back in and only removed `handler_startup`, which we no longer call after moving from vetiver to raw Plumber.